### PR TITLE
added QTL option to study report and added QTL Rmd to inst/report/

### LINKED ISSUE #1260: Feature: Create QTL Report
## Feature Details

Start with straightforward report in `{gsm}` :
- site, country, and QTL report will be generated as part of the standard pipeline
- we can aggregate QTL data into a single report across studies eventually

## Pull Request
https://github.com/Gilead-BioStats/gsm/pull/1277

### PR DESCRIPTION
**Pull request by andersonjake1988**
## Overview
fix #1260 Added QTL report for the Study_Report() function

## Test Notes/Sample Code

Notes: As of right now the report returns the study ID as the title and a summary table of the QTL reports, open to suggestions on other things to include with this.

### CODE REVIEW

File: inst/report/KRIReportBySite.Rmd
Comment (mattroumaya) [inline inst/report/KRIReportBySite.Rmd:1]: I'm getting an error when trying to run the report after these changes:

```
processing file: KRIReportBySite.Rmd
  |..................................                                 |  50% [unnamed-chunk-5] 
Quitting from lines 182-186 [unnamed-chunk-5] (KRIReportBySite.Rmd)
Error in `UseMethod()`:
! no applicable method for 'filter' applied to an object of class "character"
Backtrace:
 1. gsm::MakeKRIGlossary(names(assessment), dropped[grep("kri", dropped)])
 5. dplyr::filter(., .data$workflowid %in% c(strWorkflowIDs, strDroppedWorkflowIDs))
```
Will investigate a bit more but it might have to do with workflows where nothing has been dropped.

**Review by mattroumaya — CHANGES_REQUESTED** [review]
Looking great so far @andersonjake1988! 

Let's meet to discuss the reporting changes. 

Hold off for now, but prior to merging we'll want to remove the `dev_snapshots` directory. We'll also want to add unit tests for `Augment_Snapshot()` in a separate PR.

File: inst/report/KRIReportBySite.Rmd
Comment (andersonjake1988) [reply inst/report/KRIReportBySite.Rmd:1]: Ok, so the black dropped was just a blank `named list()` so the null arguments weren't catching it, I changed the logic a bit so it should work now

**Review by mattroumaya — CHANGES_REQUESTED** [review]
Great work here @andersonjake1988!! 

The report looks awesome, and think this is good to merge after addressing the minor comments in this review

File: R/Augment_Snapshot.R
Comment (andersonjake1988) [reply R/Augment_Snapshot.R:79]: for this one I added logic to check if "/" was at the end of cPath instead, because some people may make `cPath = "~/gsm/inst/snapshots"` and others may use `cPath = "~/gsm/inst//snapshots/"`

**Review by mattroumaya — APPROVED** [review]

**Pull request merged**
